hypothesis-extra/hypothesis-pytest/src/hypothesispytest/__init__.py

```diff
@@ -19,17 +19,20 @@
 
 class StoringReporter(object):
 
-    def __init__(self):
+    def __init__(self, config):
+        self.config = config
         self.results = []
 
     def __call__(self, msg):
+        if self.config.getoption('capture', 'fd') == 'no':
+            print(msg)
```

Comment:
> This should delegate to another reporter rather than printing

That was my thought as well. But I failed to figure out how to do that properly. Capturing ``current_reporter()`` in the hookwrapper (before entering into ``with_reporter``) and passing it to ``StoringReporter`` wouldn't help, because the hookwrapper is executed way before any tests, and it would always capture ``reporting.default``.